src/js/pages/Home.js

```diff
@@ -1,31 +1,40 @@
 import React, { Component } from 'react';
+import { getRouteProps } from 'react-static';
+import { get } from 'lodash';
+
+import SecondaryContentBanner from 'js/page_sections/SecondaryContentBanner';
+import Service311 from 'js/page_sections/Service311';
 import GlobalSearch from 'js/modules/GlobalSearch';
 import Hero from 'js/modules/Hero';
+import ExternalLink from 'js/modules/ExternalLink';
 import homepageImg from 'images/lady_bird_lake.jpg';
 
-class Home extends Component {
-
-  render() {
-    const homepageImage = {
-      file: homepageImg,
-      title: 'Lady Bird Lake walking trail',
-    }
+import jsonFileData from '__tmpdata/services';
+const services311 = get(jsonFileData, "services311", null);
 
-		return (
-			<div>
-        <Hero home={true} image={homepageImage}>
-          <span className="coa-Hero__home-preheader">Welcome to</span>
-          <h2 className="coa-Hero__home-header">Austin, TX</h2>
-          <GlobalSearch placeholder="Search for services" />
-        </Hero>
-	      <section className="wrapper coa-section">
-	        <h1> Home Page </h1>️
-	        <p>just a placeholder</p>
-	      </section>
-			</div>
-    );
+const Home = ({ allServicePages }) => {
```

Comment:
allServicePages --> is this for the future? Why do we need service page data here?